### ISSUE TITLE
ETQ Instructeur / Administrateur, je souhaite être tenu au courant des nouvelles fonctionnalités

### ISSUE THREAD

**Issue by LeSim** [demarches-simplifiees/demarches-simplifiees.fr#9235]

**Comment by Bouchralazaar** [comment]
Plutôt d'accord avec @colinux , faire un peu comme crisp :
Bandeau dans le header qui s'affiche lorsqu'il y a des nouveautés. 

![Capture d’écran 2023-06-23 à 03 42 41](https://github.com/demarches-simplifiees/demarches-simplifiees.fr/assets/89633115/becddb96-27ba-4bd2-8d4d-434483ded812)

![Capture d’écran 2023-06-23 à 03 42 58](https://github.com/demarches-simplifiees/demarches-simplifiees.fr/assets/89633115/3ebdd02e-ba47-47a9-b91d-985bc318f938)

Pour moi tous les membres de l'équipe peuvent renseigner ces nouveautés.

**Comment by CamilyaDellali** [comment]
Je suis d'accord avec vos retours et l'ensemble des membres de l'équipe pourront ainsi y contribuer en documentant les nouvelles fonctionnalités . @colinux Pour les autres startups , effectivement, il me semble que " Mon service sécurisé" https://www.monservicesecurise.ssi.gouv.fr/ a également "un centre de notifications" avec un bouton comme Crisp affiché depuis l'interface.

**Comment by tchak** [comment]
J'ai une idée folle - ne pas réinventer la roue et utiliser https://announcekit.app :)
